tools/experiments/experiments.js

```diff
@@ -345,6 +345,11 @@ const EXPERIMENTS = [
     name: 'Enables scroll snap on carousel across all browsers/OSes',
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/16508',
   },
+  {
+    id: 'amp-list-resize',
```

Comment:
I think we should add it to `doc-level optin` already since there is no API change per offline discussions.

Reply:
Ack. I think we need separate PRs now for everything that touches prod or canary config for easy rollback purposes though, right?

Reply:
oh yeah, right!

Reply:
I think we were deprecating them in favor of origin trials, which we currently don't yet have an easy process for? The intention here is to allow the issue subscribers of #10230 an easy way to opt-in so we can try and troubleshoot this before a gradual experimental launch to 100%.